### PR TITLE
Fix regression of require process

### PR DESCRIPTION
**Pull request by repeatedly**

### CODE REVIEW

**Comment by repeatedly** [comment]
@tagomoris Could you check this?

File: lib/fluent/compat/detach_multi_process_mixin.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+module Fluent
+  module Compat
+    module DetachMultiProcessMixin
```
Comment (tagomoris) [inline lib/fluent/compat/detach_multi_process_mixin.rb]: There were `DetachProcessMixin` too. So it's also needed.

Comment (repeatedly) [inline lib/fluent/compat/detach_multi_process_mixin.rb]: You are right. I forgot to add it.

**Comment by repeatedly** [comment]
Add `DetachProcessMixin`

**Comment by tagomoris** [comment]
LGTM.

**Comment by repeatedly** [comment]
Mac tests are canceled by tracis-ci.
Maybe, we should move Mac environment  tests to allow_failures...

**Pull request merged**